iocore/net/SSLUtils.cc

```diff
@@ -2149,6 +2112,7 @@ ssl_callback_session_ticket(SSL *ssl, unsigned char *keyname, unsigned char *iv,
                             int enc)
 {
   SSLCertificateConfig::scoped_config lookup;
+  SSLConfig::scoped_config params;
```

Comment:
Move this inside keyblock check so that we don't try to acquire the config unless we need it.